load-tests/route_guide_bidirectional_streaming/scripts/run.sh

```diff
@@ -20,5 +20,5 @@ set -e
 source base-scenario.sh
 
 echo "----------Running load tests----------"
-./ghz --skipTLS --proto $scriptsDir/route_guide.proto --rps 500 --duration 1800s --concurrency 10 --duration-stop wait --call routeguide.RouteGuide.RouteChat --stream-call-count=5 -d '[{"location": {"latitude": 406109563, "longitude": -742186778}, "message": "m1"},{"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}, {"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}]' bal.perf.test:443 --format influx-summary --output $scriptsDir/ghz_output.csv
+./ghz --skipTLS --proto $scriptsDir/route_guide.proto --rps 500 --duration 1800s --concurrency 1 --skipFirst 1000 --duration-stop wait --call routeguide.RouteGuide.RouteChat --stream-call-count=5 -d '[{"location": {"latitude": 406109563, "longitude": -742186778}, "message": "m1"},{"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}, {"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}]' bal.perf.test:443 --format influx-summary --output $scriptsDir/ghz_output.csv
```

Comment:
```suggestion
./ghz --skipTLS --proto $scriptsDir/route_guide.proto --rps 500 --duration 1800s --concurrency 1 --skipFirst 1000 --duration-stop wait --call routeguide.RouteGuide.RouteChat --stream-call-count=5 -d '[{"location": {"latitude": 406109563, "longitude": -742186778}, "message": "m1"},{"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}, {"location": {"latitude": 411733222, "longitude": -744228360}, "message": "m2"}]' bal.perf.test:443 --format influx-summary --output $scriptsDir/ghz_output.csv
```

Shall we find the optimum concurrency number here?. For non-isolated RPCs, this could be less, but for isolated RPCs it should be more than one.

Reply:
added num of connections and concurrency as 10
could not see errors locally in both unary and bidi-streaming